src/main/java/io/confluent/connect/elasticsearch/ElasticsearchWriter.java

```diff
@@ -189,9 +189,9 @@ public ElasticsearchWriter build() {
   }
 
   public void write(Collection<SinkRecord> records) {
+
```

Comment:
We generally try to avoid unnecessary changes like adding/removing lines so that the diff is smaller and more focused.
```suggestion
```